scripts/lint/find-markdown.ts

```diff
@@ -0,0 +1,212 @@
+import * as fs from 'fs';
```

Comment:
Adapted from previous lint-markdown.js but expanded to cover if file is net new when compared to parent branch.